docs/topics/api/addons.rst

```diff
@@ -192,6 +180,76 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
 
+-------------
+Versions List
+-------------
+
+.. _version-list:
+
+This endpoint allows you to list all versions belonging to a specific add-on.
+
+.. http:get:: /api/v3/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/versions/
+
+    .. note::
+        Unlisted or non-public add-ons require authentication and either
+        reviewer permissions or a user account listed as a developer of the
+        add-on.
+
+    :query string filter: The :ref:`filter <version-filtering-param>` to apply.
+    :>json int count: The number of versions for this add-on.
+    :>json string next: The URL of the next page of results.
+    :>json string previous: The URL of the previous page of results.
+    :>json array results: An array of :ref:`versions <version-detail-object>`.
+
+.. _version-filtering-param:
+
+   By default, the version list API will only return versions with valid statuses
+   (excluding versions that have incomplete, disabled, deleted, rejected or
+   flagged for further review files) - you can change that with the ``filter``
+   query parameter, which requires authentication and specific rights depending
+   on the value:
+
+    ================  ========================================================
+               Value  Description
+    ================  ========================================================
+                 all  Show all versions attached to this add-on. Requires
+                      either reviewer permissions or a user account listed as
+                      a developer of the add-on.
+    all_with_deleted  Show all versions attached to this add-on, including
+                      deleted ones. Requires admin permissions.
+    ================  ========================================================
+
+--------------
+Version Detail
+--------------
+
+.. _version-detail:
+
+This endpoint allows you to fetch a single version belonging to a specific add-on.
+
+.. http:get:: /api/v3/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/versions/(int:id)/
+
+    .. _version-detail-object:
+
+    :>json int id: The version id.
+    :>json string edit_url: The URL to the developer edit page for the version.
+    :>json array files: Array holding information about the files for the version.
+    :>json int files[].id: The id for a file.
+    :>json string files[].created: The creation date for a file.
+    :>json string files[].hash: The hash for a file.
+    :>json string files[].platform: The :ref:`platform <addon-detail-platform>` for a file.
+    :>json int files[].id: The size for a file, in bytes.
+    :>json int files[].status: The :ref:`status <addon-detail-status>` for a file.
+    :>json string files[].url: The (absolute) URL to download a file.
+    :>json object license: Object holding information about the license for the version.
+    :>json string|object|null license.name: The name of the license.
```

Comment:
Right, maybe we should or maybe we shouldn't link to that page somehow or at least note in the description of those fields that there can be translations.